doc/dev/dev_setup.md

```diff
@@ -20,7 +20,7 @@ or execute the various commands available in the toolbox.
     cd azure-sdk-for-python
     ```
 
-2.  Create a [virtual environment](https://docs.python.org/3/tutorial/venv.html)
+2.  Create a [virtual environment][virtual_environment]
```

Comment:
You may need to add a newline or period at the end of the sentence to separate this and the next line.

(Also I'm not sure if I'm hallucinating, but it looks like you switched the numbers 2. and 3. in line 16 and 23. Although it is showing up right in the rich diff?)